.github/workflows/check.yml

```diff
@@ -0,0 +1,23 @@
+name: Go Check
+on: [push, pull_request]
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [1.20.x, 1.21.x, 1.22.x]
```

Comment:
I don't think this is necessary, but we don't have to pay for the ci minutes so I probably doesn't matter.

Reply:
I included all releases within 1 year, it was only 30 seconds one go so 1 extra minute in total so I don't think it will be significant to the GitHub free allowance.